### PR TITLE
feat: added developer options to copy user id token 

### PR DESCRIPTION
**Pull request by nivedin**
### Description
Added a feature that helps developers copy user id token directly from the app.

### Checks
- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

File: packages/hoppscotch-app/components/app/Footer.vue [hunk 1]
```diff
@@ -144,7 +144,10 @@
               blank
               @click.native="options.tippy().hide()"
             />
-            <div class="flex px-4 py-2 opacity-50">
+            <div
+              class="flex px-4 py-2 opacity-50"
+              @dblclick="currentUser !== null && showDeveloperOptionModal()"
```
Comment (AndrewBastin) [inline packages/hoppscotch-app/components/app/Footer.vue:149]: Minimize logic inside templates. Can you move this into a event handler ?

File: packages/hoppscotch-app/components/app/Footer.vue [hunk 2]
```diff
@@ -225,6 +238,8 @@ const SIDEBAR_ON_LEFT = useSetting("SIDEBAR_ON_LEFT")
 
 const navigatorShare = !!navigator.share
 
+const currentUser = useReadonlyStream(probableUser$, null)
```
Comment (AndrewBastin) [inline packages/hoppscotch-app/components/app/Footer.vue:241]: Use `currentUser$` as the stream instead.

File: packages/hoppscotch-app/helpers/actions.ts [hunk 3]
```diff
@@ -22,6 +22,7 @@ export type HoppAction =
   | "modals.search.toggle" // Shows the search modal
   | "modals.support.toggle" // Shows the support modal
   | "modals.share.toggle" // Shows the share modal
+  | "modals.developerOptions.toggle" // Shows the developer options modal
```
Comment (AndrewBastin) [inline packages/hoppscotch-app/helpers/actions.ts:25]: Eventhough we have defined this action and a action handler, we don't actively have any bindings to it, so lets just remove this action.

File: packages/hoppscotch-app/locales/en.json [hunk 4]
```diff
@@ -44,7 +44,10 @@
     "chat_with_us": "Chat with us",
     "contact_us": "Contact us",
     "copy": "Copy",
+    "copy_user_id": "Copy user id",
```
Comment (AndrewBastin) [inline packages/hoppscotch-app/locales/en.json:47]: Something better like, "Copy User Auth Token". This is not an ID, its a token foremost.

**Review by AndrewBastin — CHANGES_REQUESTED** [review]
Please look into the given comments.

**Review by AndrewBastin — APPROVED** [review]
LGTM 💯 

Please wait for @liyasthomas's clear as well.

**Review by liyasthomas — APPROVED** [review]
LGTM.

**Pull request merged**